package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "one-toggleBlockView",
+  "version": "1.0.0",
+  "description": "Onesi Toggle Block Component",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --devtool=source-map"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/juangrt/one-toggleBlockView.git"
+  },
+  "keywords": [
+    "React",
+    "GRT",
+    "Onesi"
```

Comment:
nit.  probably don't need this.  React and GRT should be enough desciption for now.